cmd/fs/ls.go

```diff
@@ -17,14 +18,19 @@ type jsonDirEntry struct {
 	ModTime time.Time `json:"last_modified"`
 }
 
-func toJsonDirEntry(f fs.DirEntry) (*jsonDirEntry, error) {
+func toJsonDirEntry(f fs.DirEntry, dir string) (*jsonDirEntry, error) {
 	info, err := f.Info()
 	if err != nil {
 		return nil, err
 	}
 
+	name := f.Name()
+	if lsAbsolute {
```

Comment:
Can you make this function not to depend on global state but instead pass this flag to the function itself?